src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -364,7 +363,7 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                 {
                     try
                     {
-                        NotifyFeaturesAboutStateChange(record, index);
+                        NotifyFeaturesAboutStateChangeAsync(record, index).GetAwaiter().GetResult();
```

Comment:
In that case I think it would be better to use `Task.Run` instead of `QueueUserWorkItem`

Reply:
I'm pretty sure you have a warning here due to ValueTask used, you need to do `.AsTask().GetAwaiter().GetResult();`

Reply:
done